pagopa/io-app
# feat: [IOCOM-447] Add local feature flag for the redesign of the PN message detail

## Short description
This PR adds a local feature flag to enable/disable the new `PnMessageDetailsScreen` component in order to redesign the screen.

## List of changes proposed in this pull request
- Added the local feature flag `newPnMessageDetailsEnabled`
- Added the `LegacyPnMessageDetailsScreen` and `LegacyPnMessageDetails` components

## How to test
Using the `io-dev-api-server`, open a PN message and check that the screen is correctly rendered
